std/conv.d

```diff
@@ -5611,6 +5611,16 @@ Params:
 
 Returns:
     a `string`, a `wstring` or a `dstring`, according to the type of hexData.
+
+See_Also:
+    While this template performs the conversion during compile time,
+    the function $(REF fromHexString, std, digest) can be used for run time
+    conversions. However, these functions are not drop-in replacements and
```

Comment:
```
Use $(REF fromHexString, std, digest) for run time conversions. Note, these functions are not drop-in replacements and have different input requirements. This template uses builtin $(LINK2 $(ROOT_DIR)spec/lex.html#hex_string, hex strings), see  $(REF fromHexString, std, digest) for its requirements.
```

Reply:
> This template uses builtin hex strings.

No, it doesn’t. But it shares their syntax.

Reply:
Nevertheless, thanks for the suggestion. I’ll incorporate it in my next revision.

Reply:
Please let me know what you think of my changes.